src/app/game/services/game-config.service.ts

```diff
@@ -6,6 +6,10 @@ export interface GameLevelConfig {
   rounds: number;
   secondsPerRound: number;
   timeToStartClassify: number;
+  defaultTimeBetweenClassify: number;
+  minimumTimeBetweenClassify: number;

```

Comment:
To deal with the never classifying of holding finger down for a long time, could consider introducing a maximumTimeBetweenClassify? So it at least classifies every 6 seconds or something?